addons/hr/tests/test_hr_employee.py

```diff
@@ -21,6 +21,34 @@ def setUp(self):
             'image_1920': False
         })
 
+    def test_employee_smart_button_multi_company(self):
+        partner = self.env['res.partner'].create({'name': 'Partner Test'})
+        company_A = self.env['res.company'].create({
+            'name': 'company_A',
+        })
+        company_B = self.env['res.company'].create({
+            'name': 'company_B',
+        })
+        company_A_id = company_A.id
+        company_B_id = company_B.id
```

Comment:
You don't use company_A or company_B you can remove these variables I think :blush: 
```py
company_A_id = self.env['res.company'].create({
            'name': 'company_A',
        }).id
company_B_id = self.env['res.company'].create({
            'name': 'company_B',
        }).id```